src/develop/components/BrickDoc.js

```diff
@@ -0,0 +1,97 @@
+
+var BrickDoc = React.createClass({
+
+  _onClick: function () {
+    // No-op
+  },
+
+  render: function() {
+    var inline =
+      "<Bricks>\n" +
+      "  <Brick>\n" +
+      "    ...\n" +
+      "  </Brick>\n" +
+      "  ...\n" +
+      "</Bricks>";
+
+    return (
+      <DocsArticle title="Brick(s)" colorIndex="neutral-3">
+
+        <p>Lay out equivalently sized Bricks of content.</p>
+        <pre><code className="html hljs xml">{inline}</code></pre>
+
+        <section>
+          <h2>Brick Options</h2>
+          <dl>
+            <dt><code>label        string</code></dt>
+            <dd>The label of the Brick.</dd>
+          </dl>
+          <dl>
+            <dt><code>route        {"{location}"}</code></dt>
+            <dd>The route that the Brick should link to.</dd>
+          </dl>
+          <dl>
+            <dt><code>width        1|2</code></dt>
+            <dd>The width of the Brick.  Defaults to <code>1</code>.</dd>
+          </dl>
+          <dl>
+            <dt><code>height       1|2</code></dt>
+            <dd>The height of the Brick.  Defaults to <code>1</code>.</dd>
+          </dl>
+          <dl>
+            <dt><code>colorIndex   {"{category}-{index}"}</code></dt>
+            <dd>The color identifier to use for the background color of the Brick.  For example: <code>"neutral-1"</code></dd>
+          </dl>
+        </section>
+
+        <section>
+          <h2>Examples</h2>
+
+          <p>These examples show a variety of different options for the Bricks.</p>
+
+          <h3>Simple</h3>
+          <Box className="example">
+            <Bricks>
+              <Brick label="First"/>
+              <Brick label="Second" />
+              <Brick label="Third" />
+            </Bricks>
+          </Box>
+          <pre><code className="html hljs xml">
+            {"<Bricks>\n  <Brick label='First' />\n  ...\n</Bricks>"}
+          </code></pre>
+
+          <h3>Varying Colors, Sizes, and Content</h3>
+          <Box className="example">
+            <Bricks>
+              <Brick label="First" colorIndex="neutral-1" width={2} height={2} route="http://www.grommet.io/docs/">
+                <Image src="img/carousel-1.png" size="medium" />
+              </Brick>
+              <Brick label="Second" colorIndex="neutral-2" width={1} height={2} />
+              <Brick label="Third" colorIndex="neutral-3">
+                <GrommetLogo size="large" />
+              </Brick>
+              <Brick label="Fourth" colorIndex="neutral-4" />
+            </Bricks>
+          </Box>
+          <pre><code className="html hljs xml">
+            {"<Bricks>\n  <Brick label=\"First\" colorIndex=\"neutral-1\" width={2} height={2} route=\"...\">\n    <Image ... />\n  </Brick>\n  <Brick label=\"Second\" colorIndex=\"neutral-2\" width={1} height={2} />\n  <Brick label=\"Third\" colorIndex=\"neutral-3\">\n    ...\n  </Brick>\n  <Brick label=\"Fourth\" colorIndex=\"neutral-4\" />\n  ...\n</Bricks>"}
```

Comment:
can we use the `jsx-to-string` project ?

See this for reference on how to refactor this doc page: https://github.com/grommet/grommet-docs/blob/master/src/develop/components/AnchorDoc.js